Creiwry/irl-meeting-app
# fix(UI): change background picture quality Landing page

### Ticket Link
https://templatethpprojetfinal.atlassian.net/browse/THPPROJ-37

### User Story
Add a button from landing to registrations#new

### What does this PR do?
Change picture quality for background

### Where should the reviewer start?
Go on landing page

### How should this be manually tested?
See the picture has changed.
**Check in Heroku that buttons are displayed in columns** (for small size screens)

### Screenshots (if appropriate)
![image](https://github.com/Creiwry/irl-meeting-app/assets/137895449/cfba83e4-c65a-4443-a811-10424eb91193)

### Are there any breaking changes?
No
